recipe/meta.yaml

```diff
@@ -1,39 +1,47 @@
-{% set version = "0.4.5" %}
+{% set version = "0.4.6" %}
 
 package:
   name: logical-unification
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/l/logical-unification/logical-unification-{{ version }}.tar.gz
-  sha256: 7c6a6c1b7c6baa0f5b9af93f06cfc8d2419b6b793346b678ed1367c05ce74558
+  sha256: 908435123f8a106fa4dcf9bf1b75c7beb309fa2bbecf277868af8f1c212650a0
   patches:
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
-  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
     - pip
-    - python >=3.6
+    - python
     - setuptools
+    - wheel
   run:
-    - python >=3.6
-    - toolz
-    - multipledispatch
+    - python
+    - versioneer
```

Comment:
`toolz` and multipledispatch are real dependencies https://github.com/pythological/unification/blob/94e489e6cf583f24a9af2312b2eefd8aa4949a4d/setup.py#L19-L21
`versioneer` is a **dev** dependency

Reply:
Thanks for the catch! You're right, this was a mere mistake.